source/Halibut/Transport/Protocol/MessageSerializer.cs

```diff
@@ -10,17 +10,14 @@ namespace Halibut.Transport.Protocol
 {
     public class MessageSerializer : IMessageSerializer
     {
-        static readonly HalibutContractResolver halibutContractResolver = new HalibutContractResolver();
-        
         readonly RegisteredSerializationBinder binder = new RegisteredSerializationBinder();
-
         readonly HashSet<Type> messageContractTypes = new HashSet<Type>();
         
         JsonSerializer CreateSerializer()
         {
             var jsonSerializer = JsonSerializer.Create();
             jsonSerializer.Formatting = Formatting.None;
-            jsonSerializer.ContractResolver = halibutContractResolver;
+            jsonSerializer.ContractResolver = HalibutContractResolver.Instance;
```

Comment:
This line is the main reason this PR exists; to reuse the cache of types we've seen before. The binder will still reject contracts we haven't explicitly allowed, but this should mean the cache can be reused between individual requests.

Reply:
The `halibutContractResolver` was already a static and doing so made a measurable improvement  https://github.com/OctopusDeploy/Halibut/pull/143 measured using the code in this https://github.com/OctopusDeploy/Halibut/pull/144 reverted PR.

Is it even faster now?

Reply:
So I think what happened was I started from a stale master, thought I was fixing things, then after a rebase turns out you'd solved the same problem already 😄 I'll close the PR. The syntactic changes are already up in another branch I'm working on.